.github/workflows/unit-tests.yml

```diff
@@ -94,6 +94,7 @@ jobs:
       - name: install requirements
         run: |
           make test-requirements
+          pip install django-simple-history==3.10.1
```

Comment:
Its a hack just to run tests. We need to upgrade this pacakge to fix 100s of tests.